pkg/reconciler/pipelinerun/resources/apply.go

```diff
@@ -315,9 +317,16 @@ func replaceParamValues(params []v1beta1.Param, stringReplacements map[string]st
 func ApplyTaskResultsToPipelineResults(
 	results []v1beta1.PipelineResult,
 	taskRunResults map[string][]v1beta1.TaskRunResult,
-	customTaskResults map[string][]v1beta1.CustomRunResult) ([]v1beta1.PipelineRunResult, error) {
+	customTaskResults map[string][]v1beta1.CustomRunResult,
+	skippedTasks []v1beta1.SkippedTask,
+	logger *zap.SugaredLogger) ([]v1beta1.PipelineRunResult, error) {
```

Comment:
nit: I think it would be a bit cleaner to get the logger from the context instead of passing it in as a param (more consistent with the rest of the codebase)

Reply:
Oh I see, thanks! Fixed!